torch/csrc/jit/pass_manager.cpp

```diff
@@ -0,0 +1,16 @@
+#include <torch/csrc/jit/pass_manager.h>
+
+namespace torch {
+namespace jit {
+
+std::vector<Pass>& getPasses() {
```

Comment:
nit: can we make this slightly less general than `getPasses()`? More like `getCustomPasses()`?